apps/drupal/webpack.drupal.shared.js

```diff
@@ -7,15 +7,16 @@ const webpack = require('webpack');
 
 const drupal = {
   entry: {
+    'drupal-jquery': [
+      path.resolve(__dirname, 'drupal-jquery.js'),
+    ],
+    'drupal-lodash': [
+      path.resolve(__dirname, 'drupal-lodash.js'),
+    ],
     'app-drupal': [
       path.resolve(__dirname, 'index.js'),
     ],
   },
-  // These will be explicitly be provided OUTSIDE the bundle via a <script> tag in the HTML
```

Comment:
do we still need these as externals? or does our main bundle just find them from the other two entry points with no issues?

Reply:
@tjheffner Correct, they should be made available to the main bundle by the two entry points with no hiccups

Reply:
I'm going to pull this down and verify.